recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - ninja
   host:
     - cudatoolkit      # [(cuda_compiler_version or "").startswith("11")]
-    - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
+    - cuda-cudart-static  # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
Think we still need the `-dev` package as the headers are there

Reply:
Does `-dev` package include the static lib?

Reply:
With `cuda-cudart-dev` specifically it contains headers, shared libraries, and static libraries

Other `-dev` packages container headers and shared libraries (though not static libraries)